sdk/jobs/automl-standalone-jobs/automl-image-classification-multiclass-batch-scoring/image-classification-multiclass-batch-scoring-non-mlflow-model.ipynb

```diff
@@ -0,0 +1,842 @@
+    "\n",
+    "Few important parameters are\n",
+    "\n",
+    "- **model**: Link to the registered model\n",
+    "\n",
+    "- **code_path**: Folder containing the scoring script\n",
+    "\n",
+    "- **scoring_script**: Path of the scoring script relative to the `code path` parameter.\n",
+    "\n",
+    "- **environment**: Environment object which would be used to create the virtual environment for inferencing. It should of type `azure.ai.ml.entities.environment` "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create a batch deployment\n",
+    "deployment = BatchDeployment(\n",
+    "    name=\"non-mlflow-deployment\",\n",
+    "    description=\"this is a sample non-mlflow deployment\",\n",
+    "    endpoint_name=batch_endpoint_name,\n",
+    "    model=registered_model.id,\n",
+    "    code_path=\"./scripts\",\n",
+    "    scoring_script=\"batch_scoring.py\",\n",
+    "    environment=env,\n",
+    "    compute=\"cpu-cluster\",\n",
+    "    instance_count=2,\n",
+    "    max_concurrency_per_instance=2,\n",
+    "    mini_batch_size=10,\n",
+    "    output_action=BatchDeploymentOutputAction.APPEND_ROW,\n",
+    "    output_file_name=\"predictions.csv\",\n",
+    "    retry_settings=BatchRetrySettings(max_retries=3, timeout=30),\n",
+    "    logging_level=\"info\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 5.4.3 Create the deployment\n",
+    "\n",
+    "Using the `MLClient` created earlier, we'll now create the deployment in the workspace. This command will start the deployment creation and return a confirmation response while the deployment creation continues."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.begin_create_or_update(deployment)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5.5 Test the endpoint with sample data\n",
+    "\n",
+    "Using the MLClient created earlier, we'll get a handle to the endpoint. The endpoint can be invoked using the invoke command with the following parameters:\n",
+    "\n",
+    "- endpoint_name - Name of the endpoint\n",
+    "\n",
+    "- input - Dataset object holding the test dataset\n",
+    "\n",
+    "- deployment_name - Name of the specific deployment to test in an endpoint"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "registered_data_asset = ml_client.data.get(name=\"fridge-items-images\", label=\"latest\")\n",
+    "\n",
+    "test_data = Input(type=AssetTypes.URI_FILE, path=registered_data_asset.id)\n",
+    "\n",
+    "# invoke the endpoint for batch scoring job\n",
+    "job = ml_client.batch_endpoints.invoke(\n",
```

Comment:
Minor: Is there a way to overwrite some parameters like "process_count_per_node" etc. like in v1 notebook ?

Reply:
Yes Phani. Thank you for mentioning this. I added this too in the notebook. Few parameters such as "instance_count" could be overwritten using "params_override" but not all.
For eg. i think the equivalent to "process_count_per_node" is "max_concurrency_per_instance", but it is not allowed in "params_override".

Reply:
Thanks for the clarification Nilesh.